src/components/EditorWindow.js

```diff
@@ -20,6 +20,6 @@ EditorWindow.propTypes = {
   updateEditor: PropTypes.func,
 };
 
-export default connect(state => ({ editorState: state.editorState }), {
+export default connect(state => ({ editorState: state.note.editorState }), {
```

Comment:
@laurelherting using multiple reducers now with `combineReducers`. So had to add the `note` namespace here since the editorState is in reducers/note.js